api/GPUExternalTexture.json

```diff
@@ -0,0 +1,105 @@
+{
+  "api": {
+    "GPUExternalTexture": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/GPUExternalTexture",
+        "spec_url": "https://gpuweb.github.io/gpuweb/#gpuexternaltexture",
+        "support": {
+          "chrome": {
+            "version_added": "113"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": "mirror",
+          "samsunginternet_android": "mirror",
+          "webview_android": "mirror"
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "expired": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/GPUExternalTexture/expired",
```

Comment:
GPUExternalTexture.expired is non standard and I don't see you creating an MDN page.

Reply:
Ah, good catch; this is one of the non-standard things I didn't end up removing. removed now.